.travis.yml

```
@@ -14,6 +14,16 @@ matrix:
   - python: '3.7'
     env: NOXSESSION="docs"
     dist: xenial    # required for Python 3.7 (travis-ci/travis-ci#9069)
+before_install:
+  # Only test with conda on one test session.
+  - if [ ${NOXSESSION} == "tests-3.7" ] then
```

As written this is invalid yml and I think causing the failure of travis runs.
```suggestion
  - |
     if [ ${NOXSESSION} == "tests-3.7" ] then
```